package.json

```diff
@@ -104,15 +104,15 @@
     "jsdoc-fresh": "^1.0.1",
     "jsdoc-region-tag": "^1.0.2",
     "linkinator": "^2.0.0",
-    "mocha": "^9.2.2",
+    "mocha": "^10.0.0",
```

Comment:
I believe this will break for Node 12 - perhaps this is a typo that isn't caught yet upstream? https://github.com/mochajs/mocha/blob/master/package.json#L47

Reply:
We did this update for `teeny-request` all the tests passed and nothing complained. I assume because it is a dev dependency.

Reply:
After some discussion I am reverting `mocha` back to 9.2.2 so it does not bite us in the future.